system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.hpp

```diff
@@ -108,15 +104,15 @@ class PeriodicMeasurementNode : public system_metrics_collector::Collector,
   /**
    * Topic used for publishing
    */
-  const std::string publishing_topic_;
+  std::string publishing_topic_;
   /**
    * The period used to take a single measurement
    */
-  const std::chrono::milliseconds measurement_period_;
+  std::chrono::milliseconds measurement_period_;
   /**
    * The period used to publish measurement data
    */
-  const std::chrono::milliseconds publish_period_;
+  std::chrono::milliseconds publish_period_;
```

Comment:
Is this used anymore?

Reply:
The value is coming from the parameters and stored here. Is your idea that the parameters values should be read at `Start()` and not stored, so that each `Stop()`->`Start()` cycle might have updated/different parameter values?

Reply:
Sorry, this is the wrong line. I meant for line 107 (```std::string publishing_topic_;```). Mainly because there is a change with the status string that does not used ```publishing_topic_```.

Reply:
Okay, I removed the `publishing_topic_` member variable.